lib/ChGovUk/Controllers/Company/FilingHistory.pm

```diff
@@ -38,7 +38,8 @@ sub view {
     my $category_filter  = $self->get_filter('fh');                 # List of categories to filter by (optional)
     my @filter_categories = split ',', $category_filter;
 
-    my $unavailable_date = $self->config->{unavailable_date} || '2003-01-01';
+    my $unavailable_date = $self->config->{unavailable_date} || '2003-03-01';
+    my $un_date = '2003-03-01';
```

Comment:
Looks good - only suggestion would be to rename this so its more meaningful. Maybe `$request_document_unavailable_date`